projects/sandbox/src/app/dataset-info/dataset-info.component.html

```diff
@@ -181,6 +228,14 @@ <h2>
                 <ng-container *ngTemplateOutlet="tmpProcessingError"></ng-container>
                 <ng-container *ngTemplateOutlet="tmpLinkPublished"></ng-container>
               </li>
+              <li *ngIf="publishUrl" class="grid-label hide-desktop">
+                Debias
+              </li>
+              <li *ngIf="publishUrl" class="align-right hide-desktop">
+                <ng-container
+                  *ngTemplateOutlet="tmpCtrlDebias; context: { runEnabled: canRunDebias }"
```

Comment:
same here what means tmpCtrlDebias?

Reply:
`tmp` is a prefix for 'template' and `Ctrl` is 'control' - it's a template for the control to run the debias check.

The control actually appears twice (on narrow viewports it lives in the dataset-info dropdown panel) so that's why it's in a template.

Reply:
got it.